main.js

```diff
@@ -0,0 +1,24 @@
+document.getElementById("menu-btn").addEventListener("click", function () {
+  let menuList = document.getElementById("menu-list");
+  menuList.classList.remove("show-menu");
+});
+
+document.getElementById("btn-hide").addEventListener("click", function () {
+  let menuList = document.getElementById("menu-list");
+  menuList.classList.add("show-menu");
+});
+
+document.getElementById("mnu-port").addEventListener("click", function () {
+  let menuList = document.getElementById("menu-list");
+  menuList.classList.add("show-menu");
+});
+
+document.getElementById("mnu-about").addEventListener("click", function () {
+  let menuList = document.getElementById("menu-list");
+  menuList.classList.add("show-menu");
+});
+
+document.getElementById("mnu-contact").addEventListener("click", function () {
+  let menuList = document.getElementById("menu-list");
+  menuList.classList.add("show-menu");
+});
```

Comment:
There's a lot of repeated code in here

- [ ] Kindly refactor 
_I suggest you put all ID elements in an array and toggle the class you want to add for each of them_ 💡